priv/static/phoenix_live_view.js

```diff
@@ -2104,6 +2109,87 @@ removing illegal node: "${(childNode.outerHTML || childNode.nodeValue).trim()}"
   };
 
   // js/phoenix_live_view/rendered.js
+  var VOID_TAGS = new Set([
+    "area",
+    "base",
+    "br",
+    "col",
+    "command",
+    "embed",
+    "hr",
+    "img",
+    "input",
+    "keygen",
+    "link",
+    "meta",
+    "param",
+    "source",
+    "track",
+    "wbr"
+  ]);
+  var endingTagNameChars = new Set([">", " ", "\n", "	", "\r"]);
+  var modifyRoot = (html, attrs, clearInnerHTML) => {
+    let i = 0;
+    let insideComment = false;
+    let insideTag = false;
```

Comment:
I think this variable can be removed. You only set it to true once and then immediately break within the same clause.